### PR TITLE
adding french translation files

### PR DESCRIPTION
**Pull request by nat80**
Hi, amazon just launched Alexa in France this week,
Your application is great so I'd like to ask you if it's possible to add it to the list of skills on Amazon.fr
In the meantime I created the French translation files, if you need other information in french, you can ask me.

### CODE REVIEW

File: locales/fr-FR.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	"What would you like to do?": "Que voulez vous que je fasses ?",
+	"I couldn't work out the volume to use.": "Je n'ai pas pu calculer le volume à utiliser.",
+	"Try setting a volume between 0 and 10": "Essayez de régler le volume entre 0 et 10",
+	"You can only set the volume between 0 and 10": "Vous pouvez uniquement définir le volume entre 0 et 10",
+	"I couldn't find any connect devices, check your Alexa app for information on connecting a device": "Aucun appareil n'est connecté, vérifiez votre application Alexa pour plus d'informations sur la connexion d'un appareil",
+	"Connecting to a device using Spotify Connect": "Connexion à un appareils utilisant Spotify Connect",
+	"To add a device to Spotify Connect, log in to your Spotify account on a supported device such as an Echo, phone, or computer\nhttps://support.spotify.com/uk/article/spotify-connect/": "Pour ajouter un appareil à Spotify Connect, connectez-vous à votre compte Spotify sur un appareil compatible tel qu'un Echo, un téléphone ou un ordinateur \ nhttps: //support.spotify.com/fr/article/spotify-connect/",
```
Comment (thorpelawrence) [inline locales/fr-FR.json]: I noticed that there's an escaped space `\ ` where it looks like it should be `\n`, does it work as intended?

**Comment by thorpelawrence** [comment]
Looks good to me, thanks for your contribution; I'd be happy to add this to the store as soon as Amazon is able to approve it

File: locales/fr-FR.json (hunk 1 above)
Comment (thorpelawrence) [reply locales/fr-FR.json]: If you would be able to fix that and push, and I'd be happy to merge after that

**Comment by nat80** [comment]
Thank You @thorpelawrence , it's a pleasure to be able to contribute to this development

Here are my suggested translations :

Name: _Connect Control pour Spotify_

One sentence description: 
_Contrôlez vos appareils Spotify Connect avec Alexa_

Full description:
_Contrôlez vos appareils Spotify Connect avec Alexa
Le code source est disponible ici : https://github.com/thorpelawrence/alexa-spotify-connect
Ce skill nécessite un compte Spotify Premium pour certains appareils
AVERTISSEMENT:
Cette application utilise une API Spotify, mais n'est cependant ni homologué ni certifié ou approuvé de quelque manière que ce soit par Spotify. Spotify est la marque déposée du Groupe Spotify._

3 sample sentences:

_- Alexa, demande à Connect Control de jouer sur l'appareil 2
- Alexa, demande à Connect Control de mettre en pause
- Alexa, demande à Connect Control de passer sur l'appareil 3_

**Comment by thorpelawrence** [comment]
Thanks a lot for your help, I will try to submit this tonight but I might have to make a few adjustments to the code because Amazon have changed the developer interface since I last used it

Then it will probably take around a week to be approved to be published

Thanks again

File: locales/fr-FR.json (hunk 1 above)
Comment (nat80) [reply locales/fr-FR.json]: It's done 👍

**Comment by fablike** [comment]
Hello, not trying to push anything but... Is everything okay for the deployment in French skill store ?
Thank you for your work

**Comment by thorpelawrence** [comment]
Hi, unfortunately there was a delay in the reply from Amazon, and since I have received the feedback I have not had the time to fix it. Hopefully I will be able to work on it this weekend and resubmit.

Thanks for your patience.

**Comment by thorpelawrence** [comment]
Sorry for the late update, as there were a few problems with the approval from Amazon, but the skill should not be published in France!

**Pull request merged**